pkg/apb/client.go

```diff
@@ -149,28 +142,21 @@ func (c *Client) RunImage(
 	//"-e", fmt.Sprintf("OPENSHIFT_PASS=%s", clusterConfig.Password),
 	//spec.Name, action, "--extra-vars", string(params))
 
-	err = c.refreshLoginToken(clusterConfig)
+	err = c.Login()
```

Comment:
Unless I'm wrong, this shouldn't expire, so I should probably move it to initialization or something.